src/sentry/search/events/fields.py

```diff
@@ -112,6 +113,45 @@ def key_transaction_expression(user_id, organization_id, project_ids):
     ]
 
 
+def team_key_transaction_expression(organization_id, team_ids, project_ids):
+    if organization_id is None or team_ids is None or project_ids is None:
+        raise InvalidSearchQuery("Missing necessary meta for key transaction field.")
```

Comment:
Hmm can we reword this to be something more user friendly? Actually is it possible for a user to cause this? If one of these attributes are missing its likely a bug right?

Reply:
Yeah, if this error ever happens it should be a bug, I'll update this to 500 instead so the error is captured.